### PR TITLE
Added sub resources for usage records

### PR DESCRIPTION
**Pull request by senthgit**
Opening it  up for review

### CODE REVIEW

File: src/main/java/com/twilio/sdk/resource/instance/Account.java [hunk 1]
```diff
@@ -846,6 +847,181 @@ public UsageRecordList getUsageRecords() {
 	}
 
 	/**
+	 * Returns multiple Usage Records for each usage category,
+	 * each representing usage over a daily time-interval
+	 *
+	 * @return list of Usage Records over a daily time-interval
+	 */
+	public UsageRecordList getDailyUsageRecords() {
```
Comment (jingming) [inline src/main/java/com/twilio/sdk/resource/instance/Account.java]: for curiosity sake, why do we need one per time frame as opposed to a generic method where the user can specify the time frame? 

even if we leave these, can we do `getUsageRecordsForPeriod(Map<String, String> filters, Type type)` and have all these call into that one?

Comment (senthgit) [inline src/main/java/com/twilio/sdk/resource/instance/Account.java]: these are convenience methods. saves some typing for sdk users.

Comment (jingming) [inline src/main/java/com/twilio/sdk/resource/instance/Account.java]: ok sounds good!

I'll admit to this being a total nitpick, but this still looks like a lot of copy pasted code. Why not have one generic method  (this one can be `private` if we want our users to use these) that all 14 of these call into?

**Comment by jingming** [comment]
otherwise, +1

File: src/main/java/com/twilio/sdk/resource/instance/Account.java (hunk 1 above)
Comment (senthgit) [inline src/main/java/com/twilio/sdk/resource/instance/Account.java]: yeah I agree with it. its a lot of duplicated code. It makes sense to replace them all with only two methods.

**Comment by jingming** [comment]
+1

**Pull request merged**